controlpanel/frontend/static/javascripts/modules/confirm.js

```diff
@@ -16,7 +16,9 @@ moj.Modules.jsConfirm = {
       }
     });
 
-    $(document).on('click', `input[type="submit"].${this.confirmClass}`, (e) => {
+    // works on any children of a `<form>` with `confirmClass` but it's
+    // usually used on `<input type="submit">` or `<button>`
+    $(document).on('click', `form > .${this.confirmClass}`, (e) => {
```

Comment:
Not sure if this is meant to be in this PR?

Reply:
I almost used this module and found out there was a little bug (not working with buttons) and fixed it as it was a small fix.